src/model/quotes.js

```diff
@@ -79,8 +79,9 @@ class QuotesModel {
       throw ErrorHandler.CreateInternalServerFSPIOPError('Missing quoteRequest', null, fspiopSource)
     }
 
-    await this.db.getParticipant(fspiopSource)
-    await this.db.getParticipant(fspiopDestination)
+    // TODO: enums needed
```

Comment:
Would you add there to an enum and use it in the function call as it it used in many places

Reply:
Ok, let me look at cs-shared to see if there is an enum I can use here.